### PR TITLE
Add args to deployment template

### PR DESCRIPTION
**Pull request by flashpaper12**
#### What this PR does / why we need it:
This is so that when overriding the entry point, different args can be passed in

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [ ] Chart Version bumped
- [ ] Variables are documented in the README.md
- [ ] Title of the PR starts with chart name (e.g. `[pgadmin4]`)

### CODE REVIEW

File: charts/pgadmin4/templates/deployment.yaml [hunk 1]
```diff
@@ -72,11 +72,12 @@ spec:
           securityContext: {{- omit .Values.containerSecurityContext "enabled" | toYaml | nindent 12 }}
         {{- end }}
         {{- if .Values.command }}
-          command: {{ .Values.command }}
+          command:
+            {{- toYaml .Values.command | nindent 12 }}
         {{- end }}
         {{- if .Values.args }}
-          args: {{ .Values.args }}
-        {{- end }}
```
Comment (rowanruseler) [inline charts/pgadmin4/templates/deployment.yaml]: this is still required, since the if statement remains

Comment (flashpaper12) [reply charts/pgadmin4/templates/deployment.yaml]: I think you may be commenting on an outdated version. The if statement exists in the PR

**Review by rowanruseler — APPROVED** [review]

**Comment by flashpaper12** [comment]
Thanks for the approval! Is there anything else that I should do to get this PR merged?

**Comment by rowanruseler** [comment]
> Thanks for the approval! Is there anything else that I should do to get this PR merged?

yes, please bump the chart version in the chart.yaml file. Thank you for your PR.

**Comment by flashpaper12** [comment]
Done!

**Review by rowanruseler — APPROVED** [review]

**Comment by flashpaper12** [comment]
Thank you for merging!

**Pull request merged**